.vscode/settings.json

```diff
@@ -1,10 +1,13 @@
 {
-    "explorer.sortOrder": "mixed",
-    "editor.defaultFormatter": "esbenp.prettier-vscode",
-    "editor.formatOnSave": true,
-    "files.watcherExclude": {
-        "**/.git": true,
-        "**/node_modules/*": true,
-        "**/node_modules/@dydxprotocol/v4-abacus": false
-    }
+  "explorer.sortOrder": "mixed",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "files.watcherExclude": {
+    "**/.git": true,
+    "**/node_modules/*": true,
+    "**/node_modules/@dydxprotocol/v4-abacus": false
+  },
```

Comment:
all net new code is below this line. above this is just an autolint moving our tabs from 4 spaces to 2